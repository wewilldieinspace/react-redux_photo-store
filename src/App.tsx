import React from 'react'


export const App = () => {
    return (
        <h1>It works!</h1>
    )
}
