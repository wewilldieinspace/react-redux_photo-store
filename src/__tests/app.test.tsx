import React from 'react'
import renderer from 'react-test-renderer'
import { App } from '../App'


describe('Render App', () => {
    it('Render App', () => {
        const tree = renderer
            .create(
                <App />
            ).toJSON()

            expect(tree).toMatchSnapshot()
    })
})