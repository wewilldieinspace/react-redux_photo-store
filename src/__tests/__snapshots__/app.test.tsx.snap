// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App Render App 1`] = `
<h1>
  It works!
</h1>
`;
